### ISSUE TITLE
Does not parse redirection after 'echo'

### ISSUE THREAD

**Issue by andrewferrier** [bash-lsp/bash-language-server#261]
Thanks for your work on `bash-language-server`, it's great.

I get a 'Failed to parse expression' feedback on:

        echo >&2 'xyz'

(when it is valid bash, and in fact preferred by [shfmt](https://github.com/mvdan/sh)). I do not get an error on:

        >&2 echo 'xyz'

**Comment by skovhus** [comment]
This seems to have been fixed in the newer version of the grammar.

**Comment by andrewferrier** [comment]
Agreed, this no longer seems to be an issue on the most recent version. Thanks!
